### PR TITLE
Remove examples section

### PR DESCRIPTION
**Pull request by curquiza**

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -103,9 +99,9 @@ func main() {
 }
 ```
 
-With the `updateId`, you can check the status (`processed` or `failed`) of your documents addition thanks to this [method](#update-status).
+With the `updateId`, you can check the status (`processed` or `failed`) of your documents addition thanks to this [method](https://docs.meilisearch.com/references/updates.html#get-an-update-status).
```
Comment (eskombro) [inline README.md:102]:
```suggestion
With the `updateId`, you can check the status (`processed`, `enqueued` or `failed`) of your documents addition thanks to this [method](https://docs.meilisearch.com/references/updates.html#get-an-update-status).
```

File: README.md [hunk 2]
```diff
@@ -146,139 +142,26 @@ JSON output:
...
-updateRes, err = client.Documents("books").Delete("123")
-// Delete several documents
-updateRes, err = client.Documents("books").Deletes([]string{"123", "456"})
-// Delete all documents /!\
-updateRes, err = client.Documents("books").DeleteAllDocuments()
-```
+#### Custom Search <!-- omit in toc -->
 
-### Update status
+All the supported options are described in [this documentation section](https://docs.meilisearch.com/guides/advanced_guides/search_parameters.html).
```
Comment (eskombro) [inline README.md:147]:
```suggestion
All the supported options are described in [the search parameters documentation section](https://docs.meilisearch.com/guides/advanced_guides/search_parameters.html).
```

File: README.md [hunk 3]
```diff
@@ -298,6 +181,23 @@ resp, err := client.Search(indexUID).Search(meilisearch.SearchRequest{
...
+MeiliSearch documentation provides **examples** and a detailed explanation of every one of its features and functionalities, including examples on how to implement them **using this SDK**.
+
+Please read the [guides available in the documentation](https://docs.meilisearch.com/guides/) or check the [API references](https://docs.meilisearch.com/references/) to find the one that you need!
+
+The following sections may interest you:
+
+- [Manipulate documents](https://docs.meilisearch.com/references/documents.html)
+- [Search](https://docs.meilisearch.com/references/search.html)
+- [Manage the indexes](https://docs.meilisearch.com/references/indexes.html)
+- [Configure the index settings](https://docs.meilisearch.com/references/settings.html)
```
Comment (eskombro) [inline README.md:199]:
```suggestion
- **Manipulate documents**: see the [API reference](https://docs.meilisearch.com/references/documents.html) or follow our [guide](https://docs.meilisearch.com/guides/main_concepts/documents.html)
- **Search**: see the [API reference](https://docs.meilisearch.com/references/search.html) opr follow our [guide](https://docs.meilisearch.com/guides/main_concepts/search.html)
- **Manage the indexes**: see the [API reference](https://docs.meilisearch.com/references/indexes.html) or follow our [guide](https://docs.meilisearch.com/guides/main_concepts/indexes.html)
- **Configure the index settings**: see the [API reference](https://docs.meilisearch.com/references/settings.html) our [guide on relevancy](https://docs.meilisearch.com/guides/main_concepts/relevancy.html) 
```

**Review by eskombro — COMMENTED** [review]
WDYT if we link at the same time guides and references in the Documentation and examples section?

File: README.md (hunk 3 above)
Comment (curquiza) [reply README.md:199]: Despite some typos, I like this idea, we will discuss it in meetings tomorrow 🙂 (I know Charlotte wants to talk about this section)

File: README.md (hunk 2 above)
Comment (curquiza) [reply README.md:147]:
```suggestion
All the supported options are described in [the search parameters section](https://docs.meilisearch.com/guides/advanced_guides/search_parameters.html).
```

Comment (curquiza) [reply README.md:147]: Your suggestion seems a little bit "heavy": WDYT of this one?
(I know I accepted yours in the Python SDK, but I realized just now I don't really like it 😅 I will change it of cours in the Python if you agree)

File: README.md [hunk 4]
```diff
@@ -103,9 +99,9 @@ func main() {
 }
 ```
 
-With the `updateId`, you can check the status (`processed` or `failed`) of your documents addition thanks to this [method](#update-status).
+With the `updateId`, you can check the status (`processed`, `enqueued` or `failed`) of your documents addition thanks to this [method](https://docs.meilisearch.com/references/updates.html#get-an-update-status).
```
Comment (curquiza) [inline README.md:102]:
```suggestion
With the `updateId`, you can check the status (`enqueued`, `processed` or `failed`) of your documents addition thanks to this [method](https://docs.meilisearch.com/references/updates.html#get-an-update-status).
```

File: README.md (hunk 2 above)
Comment (eskombro) [reply README.md:147]: Sure, it's lighter. Even if it doesn't specify what it is a 'section' of. But I still prefer it to the first one, so go! :)

Comment (eskombro) [reply README.md:147]: Or 

```suggestion
All the supported options are described in [the search parameters documentation](https://docs.meilisearch.com/guides/advanced_guides/search_parameters.html).
```

File: README.md (hunk 3 above)
Comment (eskombro) [reply README.md:199]: I think I corrected all the typos ;)
Let's discuss it then!

File: README.md (hunk 2 above)
Comment (curquiza) [reply README.md:147]:
```suggestion
All the supported options are described in [the search parameters section](https://docs.meilisearch.com/guides/advanced_guides/search_parameters.html) of the documentation.
```

Comment (eskombro) [reply README.md:147]:
```suggestion
All the supported options are described in the [search parameters](https://docs.meilisearch.com/guides/advanced_guides/search_parameters.html) section of the documentation.
```

File: README.md (hunk 3 above)
Comment (curquiza) [reply README.md:199]: API reference -> API references

**Comment by curquiza** [comment]
Closed in favor of #85

**Pull request closed without merging**